### PR TITLE
ref(swagger):move the client code into client stubs

### PR DESCRIPTION
**Pull request by kmala**

### CODE REVIEW

**Comment by arschles** [comment]
Moving out of RC1, as this is not essential for the RC

File: data/available_versions.go [hunk 1]
```diff
@@ -42,44 +42,47 @@ func NewAvailableVersionsFromAPI(
...
-	if err != nil {
-		return []types.ComponentVersion{}, err
+		cv := new(models.ComponentVersion)
+		cv.Component = &models.Component{}
+		cv.Version = &models.Version{}
+		cv.Component.Name = component.Component.Name
+		if strings.Contains(component.Version.Version, "stable") {
+			cv.Version.Train = "stable"
+		} else {
+			cv.Version.Train = "beta"
```
Comment (vdice) [inline data/available_versions.go]: do we need allowance for train of `"alpha"` and or any others?

Comment (kmala) [inline data/available_versions.go]: currently there is no alpha version release and this need to be handled in a different way than getting from the version....https://github.com/deis/workflow-manager/issues/61

File: data/cluster_id.go [hunk 2]
```diff
@@ -50,7 +51,7 @@ func NewClusterIDFromPersistentStorage(sgc KubeSecretGetterCreator) ClusterID {
 // Get is the ClusterID interface implementation
 func (c clusterIDFromPersistentStorage) Get() (string, error) {
 	secret, err := c.secretGetterCreator.Get(wfmSecretName)
-	if err != nil {
+	if err != nil && !apierrors.IsNotFound(err) {
```
Comment (arschles) [inline data/cluster_id.go]: can you add a comment here explaining why this is here? I believe I accidentally removed the equivalent logic a few weeks ago. also, if there's no way to add appropriate tests for this, can you add an issue to add them later?

**Pull request merged**